enterprise/ha/src/main/java/org/neo4j/kernel/impl/ha/ClusterManager.java

```diff
@@ -132,7 +145,7 @@
     private final StoreDirInitializer storeDirInitializer;
     LifeSupport life;
 
-    public ClusterManager( Provider clustersProvider, File root, Map<String,String> commonConfig,
+    public ClusterManager( Provider clustersProvider, File root, Map<String,IntFunction<String>> commonConfig,
```

Comment:
Actually, a small change because there are very few usages of those constructors. I'll do that.